### PR TITLE
Add api to create public doc from private doc.

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

**Comment by msporny** [comment]
LGTM

File: lib/did-veres-one.js [hunk 1]
```diff
@@ -170,6 +170,47 @@ api.generate = util.callbackify(async function(
   return {publicDidDocument, privateDidDocument};
 });
 
+/**
+ * Convert a private key to a public key
+ */
+api.publicDidDocument = util.callbackify(async ({privateDidDocument}) => {
```
Comment (dlongley) [inline lib/did-veres-one.js]: I seem to remember that you can't use an arrow function with `util.callbackify`.

File: lib/did-veres-one.js [hunk 2]
```diff
@@ -170,6 +170,47 @@ api.generate = util.callbackify(async function(
...
+  };
+  const suites = ['authentication', 'grantCapability', 'invokeCapability'];
+  for(const suite of suites) {
+    publicDidDocument[suite] = [];
+    for(const item of privateDidDocument[suite]) {
+      const pubItem = {
+        type: item.type,
+        publicKey: []
+      };
+      for(const field of ['@context']) {
```
Comment (dlongley) [inline lib/did-veres-one.js]: Is there a reason why this is a loop?

Comment (davidlehn) [reply lib/did-veres-one.js]: I think there were some other items in that array that got taken out.

**Pull request merged**